src/NewProject/ProjectManager/project_configuration.cpp

```diff
@@ -7,17 +7,17 @@ using namespace FOEDAG;
 ProjectConfiguration::ProjectConfiguration(QObject *parent)
     : ProjectOption(parent) {
   initProjectID();
-  m_projectType = "";
+  m_projectType = 0;  // RTL
```

Comment:
can we make project type member an enum?

Reply:
We have it, but not here. I don't want to spread dependencies.